devel/tree-sitter-cpp/Portfile

```diff
@@ -3,6 +3,9 @@
 PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           tree_sitter 1.0
+PortGroup           legacysupport 1.1
+
+legacysupport.newest_darwin_requires_legacy 10
```

Comment:
for clarity, please add a comment for which symbol this is needed.

Reply:
it’s also going to be needed up to darwin 14 I think, as static_assert was added in the 10.11 SDK.

Reply:
Got it - I updated the commit to incorporate both of these fixes.